SBTi/target_validation.py

```diff
@@ -7,22 +7,23 @@
 from SBTi.configs import PortfolioAggregationConfig
 import logging
 
+from SBTi.interfaces import IDataProviderTarget, IDataProviderCompany, EScope, ETimeFrames
+
 
 class TargetValidation:
```

Comment:
hmm, I think this is a bit confusing. You call it target validation but you're also transforming (and merging) the data into a dataframe. That should probably be a separate step and validation should just be validation and return a boolean & appropriate error messages (that can be directly used in the code or API). What do you think?